README.md

```diff
@@ -170,6 +170,22 @@ strategies:
     enabled: true
 ````
 
+### RemovePodsHavingTooManyRestarts
+
+This strategy makes sure that pods having too many restarts are removed from nodes. For example a pod with EBS/PD that can't get the volume/disk attached to the instance, then the pod should be re-scheduled to other nodes.
+
+```
+apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+strategies:
+  "RemovePodsHavingTooManyRestarts":
+     enabled: true
+     params:
+       podsHavingTooManyRestarts:
+         podRestartThresholds: 100
```

Comment:
In my opinion it makes more sense for this to be singular instead of plural. Can you change this to `podRestartThreshold`?

Reply:
+1 for `podRestartThreshold`